testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -25,9 +28,59 @@ const STATUS_RESP_TRUE: &str = "success";
 const STATUS_RESP_NOT_COMMITTED: &str = "abort_by_response";
 const STATUS_RESP_POST_CONDITION: &str  = "abort_by_post_condition";
 
+pub const PATH_MEMPOOL_TX_SUBMIT: &str = "new_mempool_tx";
+pub const PATH_BLOCK_PROCESSED: &str = "new_block";
```

Comment:
Is there a document somewhere about all the sidecar's RPC paths?  Is the API stable?

Reply:
I do not believe the event dispatcher's API is documented, no.

Reply:
Added in e1bf465